custom/icds_reports/tasks.py

```diff
@@ -1859,3 +1860,8 @@ def update_service_delivery_report(target_date):
 def update_bihar_api_table(target_date):
     current_month = force_to_date(target_date).replace(day=1)
     BiharAPIDemographics.aggregate(current_month)
+
+
+def update_child_vaccine_table(target_date):
```

Comment:
Where are you aggregating this?

Reply:
added

Reply:
Should this be in celery?

Reply:
@calellowitz  please check this https://github.com/dimagi/commcare-hq/pull/27286. This cannot be in celery as mbt is dependent on it. @Rohit25negi